data-sources/netlify-builds/config.yml

```diff
@@ -0,0 +1,30 @@
+id: netlify-builds
+
+displayName: Netlify Builds
+
+description: |
+  Netlify is an all-in-one platform for automating modern web projects. Replace your hosting infrastructure, continuous integration, and deployment pipeline with a single workflow. Integrate dynamic functionality like serverless functions, user authentication, and form handling as your projects grow.
+
+install:
+  primary:
+    nerdlet: nr1-install-newrelic.quickstart-installation-plan
+    nerdlet_state:
+      quickstart_id: 5861d7f5-26c0-43ad-bda8-c893c4b27b25
+      requires_account: true
+  secondary:
+    link: https://www.npmjs.com/package/@newrelic/netlify-plugin#installation
+
+keywords:
+  - infrastructure
+  - automation
+  - Netlify
+  - Jamstack
+  - RUM
+  - browser agent
+  - continuous integration
+  - continuous deployment
+  - newrelic partner
+  - featured
```

Comment:
[nit] There is no concept of "featured" for data sources, so this keyword might not make much sense here anymore. All this effects is the results returned when doing a keyword search, so no harm in leaving it, but just letting you know its not needed for the "fetaured" functionality in I/O.

Reply:
Easy enough to remove!